### PR TITLE
1410: Added PSSS Unconfirmed Weeky Report

### PR DESCRIPTION
**Pull request by rohan-bes**
### Issue https://github.com/beyondessential/tupaia-backlog/issues/1410:

### CODE REVIEW

**Review by kael89 — APPROVED** [review]
Once more - it looks awesome!! I believe the named transformations make the config much more readable

**Comment by rohan-bes** [comment]
Thanks @kael89 ! I might just hold off merging this one until the indicators are merged, so that I can test it properly 👍

**Pull request merged**
